projects/demo-quiz/description.yaml

```
@@ -0,0 +1,31 @@
+---
+name: Confidential Quiz
+authors:
+  - Oasis Protocol Foundation: https://oasisprotocol.org
+description: |
+  A simple confidential quiz dApp. The questions and (correct)
```

(optional and potentially too American) 
```suggestion
  A simple confidential quiz dApp. The questions and answer key
```